handlers.go

```diff
@@ -138,7 +138,7 @@ func (s *Server) ServeAPI() error {
 	user.Use(s.AuthRequired(util.PermLevelUser))
 	user.GET("/api-keys", withUser(s.handleUserGetApiKeys))
 	user.POST("/api-keys", withUser(s.handleUserCreateApiKey))
-	user.DELETE("/api-keys/:key", withUser(s.handleUserRevokeApiKey))
+	user.DELETE("/api-keys/:key-or-hash", withUser(s.handleUserRevokeApiKey))
```

Comment:
nitpick: can we stay consistent `key_or_hash`

Reply:
:+1: